source/code/troubleshooter/omsagent_tsg_zip/tsg_files/tsg_code/tsg_errors.py

```diff
@@ -0,0 +1,323 @@
+import copy
+import subprocess
+
+# backwards compatible input() function for Python 2 vs 3
+try:
+    input = raw_input
+except NameError:
+    pass
+
+tsg_doc_path = "/opt/microsoft/omsagent/plugin/troubleshooter/tsg_tools/Troubleshooting.md"
+
+# error info edited when error occurs
+tsg_error_info = []
+
+# list of all errors called when script ran
+err_summary = []
+
+
+
+# set of all errors which are actually warnings
+warnings = {115, 127, 151}
+
+# dictionary correlating error codes to error messages
+tsg_error_codes = {
```

Comment:
like I mentioned above, IMO ideally these would be given informative variable names, otherwise it is a bit hard to follow the code without having to reference back here frequently

Reply:
Yeah, I changed these as well to have names through the same `tsg_error_codes.py` file:
https://github.com/microsoft/OMS-Agent-for-Linux/blob/bff7c80f869d6f5a4046ed41b4bd0ed89d69219f/source/code/troubleshooter/modules/error_codes.py#L8-L12
https://github.com/microsoft/OMS-Agent-for-Linux/blob/bff7c80f869d6f5a4046ed41b4bd0ed89d69219f/source/code/troubleshooter/modules/errors.py#L21